README.md

```diff
@@ -198,20 +198,20 @@ When a customer is ready to buy a product or start a subscription, you’ll prov
 
 Clicking a checkout link takes the customer to Stripe’s hosted checkout page, where they can complete a payment using whatever methods are available and enabled in your account.
 
-To output a checkout link, use the `craft.stripeCheckoutUrl()` function:
+To output a checkout link, use the `stripeCheckoutUrl()` function:
 
 ```twig
 {% set price = product.prices.one() %}
 
 {{ tag('a', {
-  href: craft.stripe.checkout.getCheckoutUrl(
+  href: stripeCheckoutUrl(
     [
       {
         price: price.stripeId,
         quantity: 1,
       },
     ],
-    null,
+    currentUser ? currentUser.email : null,
```

Comment:
could this not just be `currentUser`? Or is this just trying to be a little more verbose for documentation purposes?

Reply:
It’s because `currentUser` will resolve to the username.